src/wat-writer.cc

```diff
@@ -1122,7 +1165,6 @@ void WatWriter::BuildExportMaps() {
       }
 
       case ExternalKind::Except:
-        WABT_FATAL("BuildExportMaps(except) not implemented");
```

Comment:
This will be needed when you add inline exports for exceptions, I think. May be good to add a TODO